Fails to build on macOs arm PC

build fails with:
```
error[E0433]: failed to resolve: use of undeclared crate or module `async_compat`
   --> /Users/stephan/.cargo/registry/src/index.crates.io-6f17d22bba15001f/bevy_mod_reqwest-0.14.2/src/lib.rs:104:33
    |
104 |                         let r = async_compat::Compat::new(async {
    |                                 ^^^^^^^^^^^^ use of undeclared crate or module `async_compat`
```

which is because we only add `async_compat` as a dependency if on `x86`. will open a PR

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -34,5 +34,5 @@ crossbeam-channel = "0.5.7"
 # bevy-inspector-egui = {version = "0.23", git = "https://github.com/Aztro-dev/bevy-inspector-egui.git" }
 bevy = {version = "0.13.0", features = ["multi-threaded"]}
 
-[target.'cfg(target_arch = "x86_64")'.dependencies]
+[target.'cfg(not(target_arch = "wasm32"))'.dependencies]
 async-compat = "0.2.1"

